serve-soup-inventory-mgt/src/App.js

```diff
@@ -1,9 +1,70 @@
 import React from 'react';
-import './App.css';
+import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
+import pt from 'prop-types';
+
+import LoginForm from './components/organisms/LoginForm';
+import SignUpForm from './components/organisms/SignUpForm';
+import Header from './components/templates/Header';
+import InventoryGrid from './components/pages/InventoryGrid';
+import InventoryItemDetails from './components/pages/InventoryItemDetails';
+import InventoryItemForm from './components/pages/InventoryItemForm';
 
 function App() {
+  const PrivateRoute = ({ component: Component, ...rest }) => (
```

Comment:
You could have a PrivateRoute file to keep App.js cleaner and easier to read

Reply:
I'll affect the changes now